media-libs/mpvqt/mpvqt-1.0.0.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.245.0
+QTMIN=6.5.0
+inherit ecm kde.org
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${PN}/${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="MpvQt is a libmpv wrapper for QtQuick2 and QML"
```

```suggestion
DESCRIPTION="libmpv wrapper for QtQuick2 and QML"
```

This one I will fix for you on merge.